src/templates/text-box/text-box.test.js

```diff
@@ -1,7 +1,7 @@
 describe('Make sure text-box loads', () => {
   it('Should have headline, text and background-image', () => {
     cy.visit('http://localhost:3000');
-    cy.get('#debug-bar-fixture-1').click();
+    cy.get('select').select('slide: text-box top');
```

Comment:
It's better to use a class or ID as selector, since other 'select' elements might exist.
This applies to all get('select') in the file.